app.yaml

```diff
@@ -217,5 +213,4 @@ skip_files:
 - third_party/static/nginfinitescroll-1.0.0
 - third_party/static/perfectscrollbar-0.6.16
 - third_party/static/rangy-1.3.0
-- third_party/static/textAngular-1.4.5
```

Comment:
FYI we no longer use textAngular. So I got rid of it.

Reply:
Yay!